script.js

```diff
@@ -0,0 +1,105 @@
+const store = {
+  object1: {
+    name: 'Tonic',
+    img: 'image/Snapshoot_Portfolio.svg',
+    companyName: 'CANOPY',
+    job: 'Back End Dev',
+    year: 2015,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    tags: ['html', 'css', 'javaScript'],
+  },
+
+  object2: {
+    name: 'Multi-Post Stories',
+    img: 'image/Snapshoot_Portfolio(3).svg',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing.',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+
+  object3: {
+    name: 'Facebook 360',
+    img: 'image/Snapshoot_Portfolio(2).svg',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing.',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
```

Comment:
* The popup design for mobile is different from the original design kindly compare and make necessary changes 
* ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/128408578-5a650f3b-dfef-446d-b630-dad89ac541e9.png)
* YOURS : 
![image](https://user-images.githubusercontent.com/52497246/128408613-1349e501-8ff1-40d1-b6ec-7a0c0a1668ba.png)

Reply:
This issue is not properly resolved

Reply:
This issue is well addressed ✔️